packages/tinacms/src/auth/TinaCloudProvider.tsx

```diff
@@ -95,12 +96,12 @@ export const AuthWallInner = ({
               action: async () => {
                 // This does not work it looks like we have somehow getting two contexts
                 // console.log({ setEdit })
-                // setEdit(false)
-                // setActiveModal(null)
+                setEdit(false)
+                setActiveModal(null)
 
                 // This is a temp fix
-                setEditing(false) // set editing just sets the local storage
-                window.location.reload()
+                // setEditing(false) // set editing just sets the local storage
+                // window.location.reload()
```

Comment:
Minor nitpick, but I think it might be safe to just remove the commented lines in this PR.